cirq-core/cirq/protocols/json_test_data/GateFamily.json

```diff
@@ -4,7 +4,9 @@
     "gate": "XPowGate",
     "name": "Type GateFamily: cirq.ops.common_gates.XPowGate",
     "description": "Accepts `cirq.Gate` instances `g` s.t. `isinstance(g, cirq.ops.common_gates.XPowGate)`",
-    "ignore_global_phase": true
+    "ignore_global_phase": true,
+    "tags_to_accept": [],
+    "tags_to_ignore": []
```

Comment:
don't modify existing serializations and rather add new serialization tests to test the updated logic while maintaining backwards compatibility. See the comment above.